gensim/models/ldamodel.py

```diff
@@ -14,9 +14,12 @@
 
 The core estimation code is based on the `onlineldavb.py script
 <https://github.com/blei-lab/onlineldavb/blob/master/onlineldavb.py>`_, by
-`Matthew D. Hoffman, David M. Blei, Francis Bach:
-Online Learning for Latent Dirichlet Allocation, NIPS 2010
-<https://papers.neurips.cc/paper/2010/file/71f6278d140af599e06ad9bf1ba03cb0-Paper.pdf>`_.
+Matthew D. Hoffman, David M. Blei, Francis Bach:
+`'Online Learning for Latent Dirichlet Allocation', NIPS 2010`_.
+
+.. _'Online Learning for Latent Dirichlet Allocation', NIPS 2010: online-lda_
+.. _'Online Learning for LDA' by Hoffman et al.: online-lda_
```

Comment:
I think defining it one file and then referencing it from other files should work. Defining the same label in multiple files is likely to raise and error or even a warning when building the docs.

Have you tried building these with Sphinx?

Reply:
I tried defining the link reference in ldamodel.py only but this does not work.
I build the docs using `make -C docs/src html` and it works without errors or warnings.